app/javascript/dashboard/assets/scss/_foundation-custom.scss

```diff
@@ -17,9 +17,10 @@
 }
 
 .tooltip {
+  background-color: #000000d5;
   border-radius: $space-smaller;
   font-size: $font-size-mini;
-  max-width: 15rem;
+  max-width: 20rem;
```

Comment:
Are there token variables that can be used instead?

Reply:
No, we don't have it